### DISCUSSION TITLE
Greetings API with I18n language support

### DISCUSSION THREAD

**Discussion by kt-12** [js-fns/date-fns#2843] [Ideas]
It has been while I am searching for a way to greet my client (Good Morning, Good Evening etc )  based on their system timing and the language of the site they are viewing.  

So, 
* I can pass in the language code and the function will return me the greeting text based on the system time. 
* I can pass in the language code and time ant the function will return a greeting text.

**Comment by tan75** [comment]
hi @kt-12 
this is a good idea! I am going to convert this to a discussion.

**Reply by kt-12** [reply]
Great to see this getting picked up even after 4 years. I'll definitely contribute.
